packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -96,6 +101,21 @@ export interface OdspErrorResponse {
     };
 }
 
+/** Redirect error from ODSP  */
+export class OdspRedirectError extends LoggingError {
```

Comment:
I think is should implemet IFluidErrorBase too in declaration.

Reply:
Good catch.  Basically this will spell-check `errorType` for you, I don't think you'll have to make any actual changes but for sure it belongs like he said.